test/wpt/test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+// Flags: --expose-internals --experimental-worker --expose_gc
+// Note:
+// - The --expose-internals flag is specific to Node.js core's
+//   internal error system and is unnecessary for external users.
+// - The --experimental-worker flag is used to inform the
+//   WPT harness that we are running tests in the Worker scope
+//   so it doesn't use document.methods().
```

Comment:
Maybe it's only useful in some environments? On my machine I tried to remove the flag and got the same result.

Reply:
You're right. Workers are now enabled by default in Node, so we no longer need this flag. I'll remove it. 🙂